dynamo_client.py

```diff
@@ -1,44 +1,120 @@
 from datetime import datetime
 from boto3.dynamodb.conditions import Key
 import boto3
-
+from decimal import Decimal
+import json
 
 class DynamoClient:
     def __init__(self):
+        # client = boto3.resource('dynamodb', region_name='eu-west-2', aws_access_key_id=aws_access_key_id, aws_secret_access_key=aws_secret_access_key)
         client = boto3.resource('dynamodb')
         self.order_table = client.Table("order")
         self.cust_table = client.Table("customer")
+        self.esim_table = client.Table("esim_details")
 
     def get_customers(self, source_customer_id):
         print("source_customer_id " + str(source_customer_id))
         response = self.cust_table.query(
-            IndexName='source_customer_id-index',
-            KeyConditionExpression=Key(
-                'source_customer_id').eq(source_customer_id),
-        )
+                IndexName='source_customer_id-index',
+                KeyConditionExpression=Key('source_customer_id').eq(source_customer_id)
+            )
+        print(response)
         return response['Items']
-
+        
     def put_customer(self, customer):
-        print(customer)
         response = self.cust_table.put_item(Item={
-            'customer_id': customer.customer_id,
-            'source_name': customer.source_name,
-            'source_customer_id': customer.source_customer_id,
-            'orders': customer.orders,
+            'customer_id':customer.customer_id,
+            'source_name':customer.source_name,
+            'source_customer_id':customer.source_customer_id,
+            'orders':customer.orders,
             'upserted_at': customer.upserted_at
         })
 
         return response['ResponseMetadata']['HTTPStatusCode'] == 200
-
+    
     def put_order(self, order, customer):
         response = self.order_table.put_item(Item={
-            'order_id': order.id,
-            'source_name': order.source_name,
-            'source_order_id': order.source_order_id,
-            'customer_id': customer.customer_id,
-            'total_price': order.price,
-            'order_items': order.order_items,
-            'upserted_at': str(datetime.now())
-        })
-
+                    'order_id':order.id,
+                    'source_name':order.souce_name,
```

Comment:
is order.souce_name correct?

Reply:
yes @zaine2003  not correct will change and push.